src/Lecture2.hs

```diff
@@ -199,7 +273,10 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing list = case list of
+    [x, y] -> x < y
```

Comment:
This case is actually redundant as it's already covered by the next two

Reply:
Ah, true.